pkgs/by-name/xc/xcbeautify/package.nix

```
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, unzip, lib }:
```

```suggestion
{ lib
, stdenv
, fetchurl
, unzip
}:
```

nitpick: lib should be the first argument, other arguments should come in the order they appear in the body of the file.

---

> lib should be the first argument, other arguments should come in the order they appear in the body of the file

This paragraph was removed from docs.

One per line is nice since it makes diffs more readable.